qutebrowser/browser/commands.py

```diff
@@ -1739,6 +1741,9 @@ def fullscreen(self, leave=False):
 
         if not window.isFullScreen():
             window.state_before_fullscreen = window.windowState()
+        if enter:
+            window.setWindowState(window.windowState() | Qt.WindowFullScreen)
+            return
```

Comment:
No reason to skip the debug logging below with `--enter`, so I'd just add an `else:` for the line below and remove the `return`.